grouper/models.py

```diff
@@ -253,6 +253,10 @@ def name(self):
     def type(self):
         return "User"
 
+    @property
+    def is_enabled(self):
+        return self.enabled
```

Comment:
why not just directly reference `enabled`? if we need to wrap it later, we can always make it a property.